src/parser.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import annotations
+import requests as rq
+import json
+from bs4 import BeautifulSoup as bs
+import configuration as cf
+from transformers import pipeline
+
+question_name = "deepset/roberta-base-squad2"
+question_model = pipeline(
+    "question-answering", model=question_name, tokenizer=question_name
+)
+
+
+translation_name = "Helsinki-NLP/opus-mt-ru-en"
+translation_model = pipeline(
+    "translation", model=translation_name, tokenizer=translation_name
+)
+
+
+def pages_num(soup: bs) -> int:
+    pagination = soup.find("div", class_="PaginationWidget__wrapper-pagination")
+    paginator = pagination.find_all("a")
+    num = 0
+    for data_page in paginator:
+        if num < int(data_page["data-page"]):
+            num = int(data_page["data-page"])
+    return num
+
+
+def parce(to_file: str) -> None:
+    try:
+        f = open(to_file, "w")
+    except IOError as e:
+        return None
+
+    classifier_name = "joeddav/xlm-roberta-large-xnli"
+    classifier = pipeline("zero-shot-classification", model=classifier_name)
```

Comment:
Other models are defined globally and this model here 🧐

Reply:
Also added this model into ```load_models```